diffusers/training_example.ipynb

```diff
@@ -0,0 +1,3955 @@
+{
```

Comment:
<p>LR schedule -&gt; learning rate schedule in text (or add the acronym earlier)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>